netconf/session.go

```diff
@@ -22,11 +24,49 @@ type Session struct {
 	ErrOnWarning       bool
 }
 
+// FIXME: something like this
+var RegExpOutputOrMsgSeperator = regexp.MustCompile(beginOutput + "[E|H|K|C|S]*" + endOutput + "|" + msgSeperator)
```

Comment:
@jnpr-lwu , think it twice, the pattern doesn't match the variable/function name.
can we use `.*` here?